### DISCUSSION TITLE
replace a server HowTo

### DISCUSSION THREAD

**Discussion by mazataza** [minio/minio#12563] [Q&A] [answered]
we have 4 servers (each has 6 disk with 11T) and one of them we put offline the minio work stable and when it goes online after a while the respond of all nodes decrease and we can't upload files until now we couldn't find the root reason. i checked all disk on that server but without finding an issue.

now want to replace it with another server.
can we just stop all minio servers and add that server to the cluster and droping the server with the issue.
we do not want to copy files from the droped server (as it may have currupted data).
does this approach will work! and data will be shared to the new server!

**Comment by harshavardhana** [comment, marked as answer]
> we have 4 servers (each has 6 disk with 11T) and one of them we put offline the minio work stable and when it goes online after a while the respond of all nodes decrease and we can't upload files until now we couldn't find the root reason. i checked all disk on that server but without finding an issue.
> 
> now want to replace it with another server.
> can we just stop all minio servers and add that server to the cluster and droping the server with the issue.
> we do not want to copy files from the droped server (as it may have currupted data).
> does this approach will work! and data will be shared to the new server!

You can replace the new server add the same IP as old, with disks same and formatted with same settings. Start the server and it will be automatically healed.

**Reply by MoonJustry** [reply]
@harshavardhana  hi, is there any way to support replacing unhealthy node with new node with different IP or hostname? For some reason, the IP or hostname of the new node cannot be consistent with the failed node. thx.

**Reply by harshavardhana** [reply]
> @harshavardhana hi, is there any way to support replacing unhealthy node with new node with different IP or hostname? For some reason, the IP or hostname of the new node cannot be consistent with the failed node. thx.

You can always create an alias for the same hostname in /etc/hosts

**Reply by MoonJustry** [reply]
OK，thx，It seems to be like this.
